torch_xla/distributed/spmd/xla_sharded_tensor.py

```diff
@@ -108,6 +108,9 @@ def __new__(cls, elem: torch.Tensor, *args, **kwargs):
     r.global_tensor = elem.detach() if r.requires_grad else elem
     return r
 
+  def __torch_function__(cls, func, types, args=(), kwargs=None):
```

Comment:
```suggestion
  @classmethod
  def __torch_function__(cls, func, types, args=(), kwargs=None):
```

Reply:
ok let me update and give it another try